apps/src/StudioApp.js

```diff
@@ -783,6 +794,10 @@ StudioApp.prototype.resizeHeaders = function (fullWorkspaceWidth) {
   var toolboxWidth = 0;
   var showCodeWidth = 0;
 
+  var clearPuzzleHeader = document.getElementById('clear-puzzle-header');
+  var clearPuzzleWidth = clearPuzzleHeader
+      ? clearPuzzleHeader.getBoundingClientRect().width : 0;
```

Comment:
Nit, ? should be on the previous line. Style guide even backs me up :)
http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Code_formatting (see last section on binary and ternary operators)